testData/static_analyser/w288.nut line = (5) column = (7) : Warning: Function '(w288.nut:2)' is called with the wrong number of parameters.
local b = 1;
return fn(b);
       ^----


AN ERROR HAS OCCURRED [wrong number of parameters passed to '(w288.nut:2)' testData/static_analyser/w288.nut:2 (2 passed, at least 3 required)]

CALLSTACK
*FUNCTION [__main__()] testData/static_analyser/w288.nut line [5]

LOCALS
[b] 1
[fn] CLOSURE
[vargv] ARRAY
[this] TABLE
